woof-code/rootfs-skeleton/etc/rc.d/rc.sysinit

```diff
@@ -58,6 +58,14 @@ status_func() {
  return $1 #return with same status as entry param.
 }
 
+create_zram_swap() {
```

Comment:
perhaps we `return 1` if we run an older kernel not supporting zram here?

Reply:
zram can be builtin. The `echo` will fail, if zram wasn't loaded, and isn't builtin.